tests/components/hassio/test_http.py

```diff
@@ -75,7 +75,7 @@ async def test_forward_request_onboarded_user_unallowed_methods(
     ("bad_path", "expected_status"),
     [
         # Caught by bullshit filter
-        ("app/%252E./entrypoint.js", HTTPStatus.BAD_REQUEST),
+        ("app/%252E./entrypoint.js", HTTPStatus.INTERNAL_SERVER_ERROR),
```

Comment:
aiohttp has a bugfix for incorrectly decoding the url before passing it to the router. It no longer does this so its no longer possible